tests/shared_mocks.py

```diff
@@ -51,58 +57,93 @@ def encode(data):
 
 
 def get_mock_open(files: dict[str, str]):
+    """
+    Mock the process of open a files
+
+    :return the mocked files
+    """
+
+    # pylint: disable=unused-argument
     def open_mock(filename, *args, **kwargs):
         for expected_filename, content in files.items():
             if filename == expected_filename:
                 if content == "Exception":
-                    raise Exception()
+                    raise Exception()  # pylint: disable=broad-exception-raised
                 return mock.mock_open(read_data=content).return_value
         raise OSError("(mock) Unable to open {filename}")
 
     return mock.MagicMock(side_effect=open_mock)
 
 
 def statvfs(_):
+    """
+    TODO: proper documentation
```

Comment:
I don't think it's necessary to cover the tests with all this rigor, as we do with the code. And plz don't add a code docs with TODO add docs 😆